doc/hotspot-style.md

```diff
@@ -572,9 +572,13 @@ There are a few exceptions to this rule.
 
 * `#include <new>` to use placement `new`, `std::nothrow`, and `std::nothrow_t`.
 * `#include <limits>` to use `std::numeric_limits`.
-* `#include <type_traits>`.
+* `#include <type_traits>` with some restrictions, listed below.
 * `#include <cstddef>` to use `std::nullptr_t` and `std::max_align_t`.
 
+Certain restrictions apply to the inclusion of `<type_traits>`.
```

Comment:
There are no restrictions on the inclusion of <type_traits>.  Rather, there are restrictions on the
use of the declarations provided by <type_traits>.

Reply:
Will do, thanks